website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,545 @@
+want to use.
+
+To address this, we've added an additional command called `New Scala Project`
+that enables users to reuse the same mechanism from the comfort of your editor.
+Firstly, you can choose from existing templates, either curated by the Metals
+team, ones existing on the giter8
+[wiki page](https://github.com/foundweekends/giter8/wiki/giter8-templates) or
+alternatively an entirely custom template. Then you can choose the exact
+location for the new project and the name. Afterwards, Metals will run giter8 to
+create the project and offer to open that new project.
+
+![new-project](https://user-images.githubusercontent.com/3807253/81842954-2d1b3680-954d-11ea-934a-96fe53cafeb9.gif)
+
+The command is also available as a button in the welcome view of the Visual
+Studio Code Metals extension, which means it will be visible in case there are
+no imported projects in the current workspace.
+
+The first curated templates were choosen by the team, however we are happy to
+include more so that it's easier for people to discover new amazing libraries or
+frameworks.
+
+## Multiline string improvements
+
+There are two new quality of life improvements when it comes to multiline
+strings. Firstly, thanks to [mlachkar](https://github.com/mlachkar) we now
+automatically add `stripMargin` method invocation when using newline in a
+multiline string if that string uses `|` and has not existing `stripMargin`:
+
+![strip-margin](https://i.imgur.com/Z2Q1glN.gif)
+
+This behaviour can be disabled in Metals settings using
+`metals.enableStripMarginOnTypeFormatting`.
+
+Another useful improvement coming from [colineto](https://github.com/colineto)
+is a new code action that can easily convert single string into the traditional
+multiline string with `|` and `stripMargin` at the end:
+
+![new-action](https://i.imgur.com/5ohRcl9.gif)
+
+## Multiple workspace build tools selection
+
+Previously, Metals would only pick up a single build tool in a workspace even if
+in reality there were multiple available ones. To help with that we added a
+prompt to choose the desired build tool in workspaces with multiple build files.
+
+An example situation where this is needed is when you have a project with both a
+`build.sbt` and `build.sc`, but the `build.sc` is what you use for your build
+definition. Previously if both were detected, it was automatically considered an
+Sbt workspace, however now it's possible to choose.
+
+![multiple-build-tool](https://i.imgur.com/pAgZK7u.gif)
+
+If you choose the wrong one, or change the main build tool later on, you'll need
+to remove the .metals/ directory to reset the decisions.
+
+This great new feature was contributed by [ckipp01](https://github.com/ckipp01)!
+
+## Miscellaneous improvements
+
+- Fix run/debug in Maven workspaces.
+- Always run newest sbt version if none is specified.
+- Fix find references for Scala 3 on Windows.
+- Pick jvm targets with higher priority when running without specifying the
```

Comment:
In VS Code it can be done using launch.json. You don't have to specify the target and we will try to find one for the user. I changed it to first check for the JVM ones

Reply:
But it can be done using different editors too, so not sure how best to really explain it here. I will change `running` to `using run/debug`